platform/security/platform-security-core-api/pom.xml

```diff
@@ -203,17 +209,17 @@
                                         <limit>
                                             <counter>INSTRUCTION</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.53</minimum>
+                                            <minimum>0.56</minimum>
```

Comment:
Make sure you're still allowing for buffer space on these minimum values. These were much lower than necessary intentionally because the build fails on windows otherwise.